library/core/testing/src/testmod/java/org/quiltmc/qsl/testing/test/ExampleTestSuite.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright 2016, 2017, 2018, 2019 FabricMC
- * Copyright 2022 The Quilt Project
+ * Copyright 2023 The Quilt Project
```

Comment:
If you're updating the copyright it may be worth updating to 2024? But I'm not very versed around copyright things, decide for yourself.